core/cap-0038.md

```diff
@@ -157,7 +157,7 @@ index 0e7bc842..2c568f26 100644
 +
 +struct TrustLineEntryExtensionV2
 +{
-+    int32_t liquidityPoolUseCount;
++    int32 liquidityPoolUseCount;
```

Comment:
Same error exists in `LedgerHeaderExtensionV1`, please fix that too.

Reply:
done